.agent/rules/create-branch-on-change.md

```
@@ -0,0 +1,170 @@
+---
+trigger: always_on
+description: Automatic branch management - check current branch and create new branches based on user context
+---
+
+# Branch Management Rule
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)